_layouts/pr.html

```diff
@@ -65,5 +65,6 @@ <h1>{{ page.title }} ({{components}})</h1>
     <p>
       {{ content }}
     </p>
+    {% include backlinks.html %}
```

Comment:
The idea here is that you can reference a PR at another PR's page and it will show the backlink in the same way that is currently displayed at the topics pages. In my mind, "related meetings" could be broader than just meetings that reference each other, therefore backlinks in PRs pages is just another way to navigate around.
Based on the recent changes, if no backlinks exist, nothing will be displayed at the PR page.